src/docs/resources/faq.md

```diff
@@ -345,37 +345,44 @@ Flutter should be able to run most Dart code that does not import
 
 ### How big is the Flutter engine?
 
-In December 2018, we measured the download size of a
+In July 2019, we measured the download size of a
 [minimal Flutter app][] (no Material Components,
 just a single `Center` widget, built with
-`flutter build apk`), bundled and compressed as a
-release APK, to be approximately 4.06&nbsp;MB.
+`flutter build apk --split-per-abi`),
+bundled and compressed as a release APK,
+to be approximately 4.3 MB for ARM,
+and 4.6 MB for ARM 64.
+
+In ARM, the core engine is approximately 3.2 MB (compressed),
+the framework + app code is approximately 920.6 KB (compressed),
+the LICENSE file is 54.3 KB (compressed), necessary Java code
+(classes.dex) is 113.6 KB (compressed).
 
-For this simple app,
-the core engine is approximately 2.7&nbsp;MB (compressed),
-the framework + app code is approximately 820.6&nbsp;KB (compressed),
-the LICENSE file is 53.5&nbsp;KB (compressed),
-necessary Java code (classes.dex) is 61.8&nbsp;KB (compressed),
-and there is approximately 450.4&nbsp;KB of (compressed) ICU data.
+In ARM64, the core engine is approximately 3.5 MB (compressed),
+the framework + app code is approximately 872 KB (compressed),
+the LICENSE file is 54.3 KB (compressed), necessary Java code
+(classes.dex) is 113.6 KB (compressed).
 
 These numbers were measured using [apkanalyzer][],
-which is also built into [Android Studio][].
+which is also [built into Android Studio][].
 
-On iOS, a release IPA of the same app has a download
-size of 10.8&nbsp;MB on an iPhone X, as reported by Apple's
-App Store Connect. The IPA is larger than the APK
-mainly because Apple encrypts binaries within the IPA,
-making the compression less efficient
+On iOS, a release IPA of the same app has a download size of
+10.8 MB on an iPhone X, as reported by Apple’s App Store Connect.
```

Comment:
While Android APKs increased in size between December and July, the size of an iOS IPA stayed the same?